### ISSUE TITLE
Lexicon - Sortable table keyboard should always be visible with sufficient target size

### ISSUE THREAD

**Issue by catbusofthecanals** [jadu/pulsar#1591]
**Describe the bug**
The sortable table has keyboard accessible alternative control to sorting the table but these are not visible until tabbed to. Whilst this isn’t a fail of 2.5.7 Dragging Movements (AA) as a simple pointer alternative is available or 2.1.1 Keyboard (A) as the alternative controls are keyboard accessible, they should always be visible. Without supplementary instructions a keyboard user or user with motor impairments who would struggle to activate dragging controls would be unaware of the existence of alternative controls and may avoid interacting with table entirely.

The current alternative controls don't meet 2.5.8 Target Size (AA) requirements as they are 12 x 24 px without sufficient spacing to make up a 24 x 24 px touch target radius.

![Image](https://github.com/user-attachments/assets/2243b224-03d8-48c4-8a67-08ca6f6cacb3)

Additionally the alternative controls are currently link elements with the accessible name “Move up” or “Move down”. These controls should be semantic <button> elements as they activate controls rather than redirecting to a different page. There should also be supplementary visually hidden text to indicate which row item “Move up” or “Move down” refers to.

**To Reproduce**
1. Go to Pulsar Lexicon
2. Click on Tables then navigate to Sortable Tables
3. Navigate to the table rows by keyboard until keyboard accessible controls appear
4. Inspect to identify target size issue

**Expected behavior**
Alternative controls should always be available and visible so users are aware there are additional means of interacting with the table. The controls should meet target size requirements and be semantic button controls as they activate controls.

**Screenshots**

![Image](https://github.com/user-attachments/assets/66f4f1c9-3696-4a35-bf81-c22427cc55e5)
![Image](https://github.com/user-attachments/assets/b46aaf71-9521-4ed3-bbea-2fd5b21746ff)

**Additional context**
Related success criteria:
[2.5.8 Target Size (AA)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum)
[2.1.1 Keyboard (A)](https://www.w3.org/WAI/WCAG22/Understanding/keyboard)
[2.5.7 Dragging Movements (AA)](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements.html)

**Suggested fixes**
Ensure the keyboard and single pointer accessible alternative controls are always visible and that they meet a target size of at least 24 x 24 px. Change to <button> elements and provide supplementary text to each accessible name so screen reader users are aware which item they are interacting with eg. “Move row X up”
